semantic_conventions/faas-common.yaml

```diff
@@ -0,0 +1,77 @@
+groups:
+  - id: attributes.faas.common
+    type: attribute_group
+    brief: "Describes FaaS attributes."
+    prefix: faas
+    attributes:
+      - id: trigger
+        brief: 'Type of the trigger which caused this function invocation.'
+        type:
+          allow_custom_values: false
```

Comment:
Outside the scope of this PR, but I think we should stop using this value for a variety of reasons.  I'll write this up in an issue/proposal for review.